_ingest-pipelines/processors/pipeline.md

```diff
@@ -0,0 +1,187 @@
+---
+layout: default
+title: Pipeline
+parent: Ingest processors
+nav_order: 200
+---
+
+# Pipeline processor
+
+The `pipeline` processor defines a series of processing steps to be executed on documents during the ingestion or update process. This is particularly useful when you need to perform complex data transformations or apply multiple processing steps to your data before it is indexed.
```

Comment:
`pipeline` processor makes a pipeline can reference another pre-defined pipeline, it can be used when there's a general pipeline with some common processors which can be shared among other pipelines, users can reference the general pipeline directly without redefining the common processors.